tensorflow/compiler/xla/service/gpu/convolution_thunk.cc

```diff
@@ -33,6 +33,7 @@ using se::dnn::ConvolutionDescriptor;
 using se::dnn::DataLayout;
 using se::dnn::FilterDescriptor;
 using se::dnn::FilterLayout;
+using se::dnn::AlgorithmType;
```

Comment:
We added this "using" statement, but then we fully-qualify the type below.  We should do only one or the other.

Reply:
Fixed - now using unqualified name elsewhere in file.